### PR TITLE
[Doctrine] add `AsDoctrineListener` attribute

### PR DESCRIPTION
**Pull request by MrYamous**

### CODE REVIEW

File: doctrine/events.rst [hunk 1]
```diff
@@ -143,9 +143,28 @@ do so, define a listener for the ``postPersist`` Doctrine event::
...
+a Doctrine listener in your application::
+
+        // src/EventListener/SearchIndexer.php
+        namespace App\EventListener;
+
+        // ...
+        use App\Entity\Product;
+        use Doctrine\Bundle\DoctrineBundle\Attribute\AsDoctrineListener;
+        use Doctrine\ORM\Events;
+        use Doctrine\Persistence\Event\LifecycleEventArgs;
```
Comment (OskarStark) [inline doctrine/events.rst]:
```suggestion
        use Doctrine\Bundle\DoctrineBundle\Attribute\AsDoctrineListener;
        use Doctrine\ORM\Events;
```

File: doctrine/events.rst [hunk 2]
```diff
@@ -143,9 +143,25 @@ do so, define a listener for the ``postPersist`` Doctrine event::
...
+        use Doctrine\Bundle\DoctrineBundle\Attribute\AsDoctrineListener;
+        use Doctrine\ORM\Events;
+
+        #[AsDoctrineListener(event: Events::postPersist, priority: 500, connection: 'default')]
+        class SearchIndexer
+        {
+            // ...
+        }
+
+That's it. Alternatively, if you prefer to not use PHP attributes, you must
```
Comment (OskarStark) [inline doctrine/events.rst]:
```suggestion
Alternatively, if you prefer to not use PHP attributes, you must
```

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Is this possible since 6.2? If yes, please add a `versionadded` directive. You can search the repository for examples.

Thanks

**Comment by MrYamous** [comment]
> Is this possible since 6.2? If yes, please add a `versionadded` directive. You can search the repository for examples.
> 
> Thanks

It is possible since DoctrineBundle 2.8, I feel like it is before SF 6.2.
But seems to be compliant with 5.4 maybe I need rebase ?

**Comment by OskarStark** [comment]
Lets target `6.2` and as this is an "external" attribute, no `versionadded` is needed 👍

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Matthieu for contributing this missing feature!

**Pull request merged**